finagle-example/src/main/scala/com/twitter/finagle/example/http/HttpOpenTracingClient.scala

```diff
@@ -0,0 +1,64 @@
+package com.twitter.finagle.example.http
+
+import com.twitter.finagle.builder.ClientBuilder
+import com.twitter.finagle.http._
+import java.net.InetSocketAddress
+import com.twitter.finagle.{Service, SimpleFilter}
+import com.twitter.finagle.tracing.WireTracingFilter
+import com.twitter.io.Charsets
+import com.twitter.util.Future
+
+import scala.util.Random
+
+import com.lightstep.tracer.jre.JRETracer
+import com.lightstep.tracer.shared.Options
+
+/**
+ * An example of using Filters with Clients. Below, requests are traced using an 
+ * OpenTracing tracer and the HTTP OpenTracing client filter. Two parallel requests are
+ * made and when they both return (the two Futures are joined) the TCP connection(s)
+ * are closed. 
+ */
+object HttpOpenTracingClient {
+
+  def main(args: Array[String]) {
+
+    val clientWithoutFilter: Service[Request, Response] = ClientBuilder()
+      .codec(Http())
+      .hosts(new InetSocketAddress(8080))
+      .hostConnectionLimit(1)
+      .build()
+
+    // initialize an OpenTracing tracer and filter to trace requests
+    val tracer = new JRETracer(new Options("{your_lightstep_token}"))
+    val traceRequest = new OpenTracing.ClientFilter[Request, Response](tracer)
+
+    // Create a base span to trace all requests made by this client
```

Comment:
let's wrap 36-53 in its own standalone (blocking) function, then call it three times in a loop (or something) to clearly separate what happens once vs what happens for every request.